tests/api/test_aws.py

```diff
@@ -345,7 +415,9 @@ def test_grant_bucket_access(iam, users, resources):
 )
 def test_revoke_bucket_access(iam, users, resources):
     bucket_arn = 'arn:aws:s3:::test-bucket'
-    path_arns = [f'{bucket_arn}{resource}' for resource in resources]
+    path_arns = []
+    if resources:
+        path_arns = [f'{bucket_arn}{resource}' for resource in resources]
```

Comment:
again, probably it can be left as before without causing a problem as when `resources = []` `path_arns` should just be `[]`, don't think you need the `if`.

Reply:
👍